src/main/webapp/aboutus.html

```diff
@@ -35,11 +35,11 @@ <h2>Teammate A Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate B Name</h2>
+    <h2>Chenghan Zhou</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: Refreshing, intense, challenging</li>
+      <li>Aspirational Hobby: Create an animation</li>
+      <li>Ask me About: My cooking video channel</li>
```

Comment:
Sounds cool! Can you explain how it started and maybe share a link?

Reply:
I am interested in design and digital painting, so I want to start some my own things. I start with cooking because it's something I am familiar with, and I spent most of the time thinking about the script, topic and how to cut the video... The videos are in Chinese so maybe not easy to understand the humors when watching.